config/charts/inferencepool/values.yaml

```diff
@@ -6,7 +6,29 @@ inferenceExtension:
     tag: main
     pullPolicy: Always
   extProcPort: 9002
-  env: []
+  env:
+    # The default OTEL_* environments is used to config the behaviour of OTel SDK
+    # If you also enabled trace.autoENVInject setting, the auto env inject will be skipped by opentelemetry-operator,
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
```

Comment:
Should we inject these env vars as part of the helm template? If these are "static" config and don't require user configuration.